core/src/main/java/com/fincity/saas/core/functions/connection/rest/CallRequest.java

```diff
@@ -1,4 +1,4 @@
-package com.fincity.saas.core.functions.rest;
+package com.fincity.saas.core.functions.connection.rest;
```

Comment:
move it to com.fincity.saas.core.functions.rest, just keeping in sync with the namespace.